etc/servo_automation_screenshot.py

```diff
@@ -0,0 +1,129 @@
+import getopt
+
+
+def servoReadyToAccept(url, payload, headers):
+    while(True):
+
+        try:
+            # Before sending an additional request, we wait for one second each time
+            time.sleep(1)
+            session_request = requests.post(url, data=payload, headers=headers)
+            json_string = session_request.json()
+            # On success, we move on to render the files
+            break
+        except Exception as e:
+            time.sleep(5)
+            print('Exception: ', e)
+    return json_string
+
+
+def ensureScreenshotsDirectoryExists():
+    if not os.path.exists('screenshots'):
+        os.makedirs('screenshots')
+
+
+def renderHTMLfiles(num_of_files, url, file_url, json_string, headers, cwd):
+    for x in range(num_of_files):
+
+        json_data = {}
+        json_data['url'] = 'file://' + file_url + 'file' + str(x) + '.html'
+        print(json_data['url'])
+        json_data = json.dumps(json_data)
+        requests.post(url + '/' + json_string['value']['sessionId'] + '/url', data=json_data, headers=headers)
+        # print('Response for html file post: ',url_request)
+        screenshot_request = requests.get(url + '/' + json_string['value']['sessionId'] + '/screenshot')
+        image_data_encoded = screenshot_request.json()['value']
+        with open("screenshots/output_image_" + str(x) + ".png", "wb") as image_file:
+            image_file.write(base64.decodebytes(image_data_encoded.encode('utf-8')))
+        print("################################")
+
+        print("The screenshot is stored in the location: " + cwd +
+              "/screenshots/ with filename: output_image_" + str(x) + ".png")
+
+        print("################################")
+
+
+def main(argv):  # take inputs from command line by considering the options parameter i.e -h, -p etc.
+
+    # Local Variables
+    port = ''
+    resolution = ''
+    file_url = ''
+    num_of_files = ''
+    cwd = os.getcwd()
+    url = ''
+    payload = "{}"
+    headers = {'content-type': 'application/json', 'Accept-Charset': 'UTF-8'}
+    json_string = ''
+    try:
+        # input options defined here.
+        opts, args = getopt.getopt(argv, "hu:p:i:r:n:", ["url=", "port=", "ifile=", "resolution=", "numOfFiles="])
+    except getopt.GetoptError:
+        # an error is generated if the options provided in commandline are wrong.
+        # The help statement is printed on how to input command line arguments.
+        print('Enter the correct command to run this file')
+        sys.exit(2)
+    for opt, arg in opts:
+        if opt == '-h':  # -h means help. Displays how to input command line arguments
+            print('Enter the correct command to run this file')
+            sys.exit()
+        elif opt in ("-p", "--port"):  # store the value provided with the option -p in port variable.
+            port = arg
+        elif opt in ("-u", "--url"):  # store the value provided with the option -u in url variable.
+            url = arg
```

Comment:
Do we need this argument? Seems like url will always be localhost.

(We can also make `port` default to 8080 or something)